readthedocs/templates/homepage.html

```diff
@@ -55,7 +55,7 @@ <h2>{% trans "Technical documentation lives here" %}</h2>
 
   <section>
     <div class="feature">
-      <h3>{% trans "Free docs hosting for open source" %}</h3>
+      <h3>{% trans "Free docs hosting for open source projects" %}</h3>
```

Comment:
While adding "projects" could be slightly more accurate, the former text is also understandable and to my knowledge not incorrect. For instance, if someone asks me what I do, I will often respond "I work with open source".

"Richard Stallman has a long legacy in open source"
"[Read the Docs provides] Free docs hosting for open source"
"OpenSUSE's community love open source"
etc.

Reply:
I don't think this change to a source translation is worth breaking existing translations (seeing that this would be the outcome)

Reply:
> I don't think this change to a source translation is worth breaking existing translations (seeing that this would be the outcome)

They would then be updated...

Reply:
Yes exactly. That's the problem.